assets/js/common/CheckCustomizationModal/CheckCustomizationModal.jsx

```diff
@@ -79,10 +79,11 @@ function CheckCustomizationModal({
       </p>
       <CheckableWarningMessage
         hideCheckbox={customized}
-        warningText={checkBoxWarningText}
         checked={checked}
         onChecked={() => setChecked((prev) => !prev)}
-      />
+      >
+        <span className="font-semibold">{checkBoxWarningText}</span>
```

Comment:
What if we put the `span` in the component itself and only leave the text as children.
This way we don't need to replicate the span and classname, and after all, all the texts should have the same font in the component.
What do you think?

Reply:
Yeah sure, i was thinking between this two options :) 
I'm going to adjust like you suggest it. I was not sure if we want more options to modify or not :)